docs/root/intro/arch_overview/advanced/attributes.rst

```diff
@@ -162,7 +162,17 @@ Data exchanged between filters is available as the following attributes:
    :widths: 1, 1, 4
 
    metadata, :ref:`Metadata<envoy_v3_api_msg_config.core.v3.metadata>`, Dynamic request metadata
-   filter_state, "map<string, bytes>", Mapping from a filter state name to its serialized string value
+   filter_state, "map<string, Value>", Mapping from the filter state name to the object value
```

Comment:
Curious, since this PR introduces new attribute `upstream filter state` only, why are we updating docs for `filter_state` attribute? Is it because the doc for this attribute is out of sync with the code?